src/sources/gcp_pubsub.rs

```diff
@@ -1041,7 +1042,7 @@ mod integration_tests {
             let lines: Vec<_> = test_util::random_lines(44).take(count).collect();
             let messages: Vec<_> = lines
                 .iter()
-                .map(base64::encode)
+                .map(|input| BASE64_STANDARD.encode(input))
```

Comment:
```suggestion
                .map(BASE64_STANDARD.encode)
```

I thought this would work, but the compiler didn't like it.